### ISSUE TITLE
Exceptions during commit

### ISSUE THREAD

**Issue by AZWN** [JackAdams/meteor-transactions#72]
When an error occurs during a commit, it is logged, but swallowed( see this [line](https://github.com/JackAdams/meteor-transactions/blob/master/lib/transactions-common.js#L579))
I think they should be thrown further, or it should be configurable to swallow them.

**Comment by JackAdams** [comment]
Yes, you're probably right.  I'll need to think through the implications of this carefully first, so can't promise anything immediately.

**Comment by AZWN** [comment]
I understand :-)

Maybe it's a strange thought, but it could be implemented configurable, and off by default. In that case, for existing systems, behaviour will not change, while any user, who wants that functionality, can enable it. Somthing like this: 

```
this.rethrowCommitError = false;
...
catch (err) {
    self.log(err);
    doRollback(err);
    if (rethrowCommitError) {
        throw err; // Or a new wrapper around this error
    }
    return;
}

**Comment by JackAdams** [comment]
Just released 0.8.1 with the suggested functionality of opting in to rethrowing errors during commits.
Set `tx.rethrowCommitError = true;` (on client and server) to change default behaviour. Or set `{rethrowCommitError: true}` in the options hash when starting a transaction.

**Comment by AZWN** [comment]
Yeah, that was exactly what I intended, thnx!
